src/components/content/Privacy.tsx

```diff
@@ -532,6 +538,13 @@ class PrivacyPolicy extends React.PureComponent {
         <Typography variant="body1">
           Stand dieser Datenschutzerklärung: 31.03.2020
         </Typography>
+        <Box paddingBottom={4} marginTop={4}>
```

Comment:
this has to go outside the `<PrivacyWrapper>`, otherwise it won't be full width

Reply:
FYI you then have to wrap everything in a so-called Fragment `<></>` because you can only return a single React node.

Reply:
done